### ISSUE TITLE
There's no access to swagger 

### ISSUE THREAD

**Issue by Dean-ZhenYao-Wang** [OnefoursevenLabs/Orchard-Swagger#2]
I installed your module, but I couldn't access the swagger page after it was enabled 。

**Comment by SzymonSel** [comment]
Try restarting the app. The default route is: http://yourapp/swagger/ui/index

**Comment by Dean-ZhenYao-Wang** [comment]
I've tried, it won't be able to access the swagger page, however, the problem was solved let me in other ways, my solution is, will Swaggerconfig. Cs file and OrchardShellEvents cs file copy to Orchard. The Web project, so that it can be directly accessed through http://app/swagger swagger page. 

In addition, for the webapi under multiple namespaces, the display on the page will be repeated, 

Such as: 

API/Orchard. AAA/Sto / {id} 

The API/Orchard. BBB/Sto / {id} 

However, actually. Only API/Orchard. AAA/Sto/{id} is the correct result .

For example I solved the problem of just now, I defined a CustomApiExplorer, and use it to replace the system's own ApiExploer service, then the problem is solved, I think, can the code contributions into your this project, in addition, I need to this project is released to the Orchard of web station, let all the people who use the Orchard can download directly by Orchar background using this module ？

**Comment by Dean-ZhenYao-Wang** [comment]
I've tried, it won't be able to access the swagger page, however, the problem was solved let me in other ways, my solution is, will Swaggerconfig. Cs file and OrchardShellEvents cs file copy to Orchard. The Web project, so that it can be directly accessed through http://app/swagger swagger page.

In addition, for the webapi under multiple namespaces, the display on the page will be repeated,

Such as:

API/Orchard. AAA/Sto / {id}

API/Orchard. BBB/Sto / {id}

However, actually. Only API/Orchard. AAA/Sto/{id} is the correct result .

For example I solved the problem of just now, I defined a CustomApiExplorer, and use it to replace the system's own ApiExploer service, then the problem is solved, I think, can the code contributions into your this project, in addition, I need to this project is released to the Orchard of web station, let all the people who use the Orchard can download directly by Orchar background using this module ？
